tools/dataExtractor.php

```diff
@@ -0,0 +1,314 @@
+<?php declare(strict_types=1);
```

Comment:
shouldn't this script be moved to the `exporters` folder within the `tools` directory?